app/src/components/market/profile/view.tsx

```diff
@@ -116,6 +126,21 @@ const ViewWrapper = (props: Props) => {
               ]}
             />
           )}
+          {questionId && (
+            <TitleValue
+              title={'Realitio'}
+              value={[
+                <a
+                  href={`https://realitio.github.io/#!/question/${questionId}`}
+                  key={1}
```

Comment:
What is this `key` doing here?

Reply:
> What is this `key` doing here?

Mmm I think this `key` is because the value is an array. I'm going to remove the array and the key because we don't need it